cg/store/models.py

```diff
@@ -387,7 +387,9 @@ class Family(Model, PriorityMixin):
     customer_id = Column(ForeignKey("customer.id", ondelete="CASCADE"), nullable=False)
     customer = orm.relationship(Customer, foreign_keys=[customer_id])
     data_analysis = Column(types.Enum(*list(Pipeline)))
-    data_delivery = Column(types.Enum(*list(DataDelivery)))
+    data_delivery = Column(
+        types.Enum(DataDelivery, values_callable=lambda obj: [e.value for e in obj])
+    )
```

Comment:
what does this difference do?

Reply:
It is the same as before, but alembic was unable to autogenerate a revision for the reordering of the fields with the syntax that was used. This syntax does the same, it uses the values of the enum.

Reply:
Removed it